src/content/en/updates/2017/03/nic57.md

```diff
@@ -0,0 +1,131 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Chrome 57 for developers? Grid based layouts, Media Session API and more.
+
+{# wf_published_on: 2017-03-08 #}
+{# wf_updated_on: 2017-03-08 #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_tags: chrome57,new-in-chrome,css,progressive-web-apps,media,grids #}
+{# wf_featured_snippet: With Chrome 57, you can now use <code>display: grid</code> for grid based layouts, use the <b>media session API</b> to customize the lock screen and notifications with information about the media being played and more. Pete LePage has all the details and how you can use these new developer features in Chrome 57! #}
+
+# New In Chrome 57 {: .page-title }
```

Comment:
'In' should be lowercase. (I want to clarify that I'm not mistaking this for a level 2 or lower. The rules for title case specify that prepositions be lowercase.)